examples/shader/shader_instancing.rs

```diff
@@ -250,9 +256,11 @@ impl<P: PhaseItem> RenderCommand<P> for DrawMeshInstanced {
         item: &P,
         _view: (),
         instance_buffer: Option<&'w InstanceBuffer>,
-        (meshes, render_mesh_instances): SystemParamItem<'w, '_, Self::Param>,
+        (meshes, render_mesh_instances, mesh_allocator): SystemParamItem<'w, '_, Self::Param>,
         pass: &mut TrackedRenderPass<'w>,
     ) -> RenderCommandResult {
+        let mesh_allocator = mesh_allocator.into_inner();
```

Comment:
It might be nice to have a quick comment in the example pointing the user to further inspect the docs for `MeshAllocator` to learn more about why this is necessary.

Reply:
It's a borrow check workaround. I wrote a comment above it.